mobile-widgets/qml/Export.qml

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0
 import QtQuick 2.12
-import QtQuick.Controls 2.12
+import QtQuick.Controls 2.13
```

Comment:
why do we need 2.13? It's always nice to have a comment in the commit message that explains why we need a newer version of any dependency

Reply:
It might just be me, I am using 2.13 normally, let me test that.

Reply:
Found the problem, the ProgressBar have some problems solved in 2.13, but I have already included the workaround for 2.12

Reply:
I can never tell which version goes with which Qt version... in general, for the mobile app I'm MUCH MORE willing to require a reasonably new Qt version. Being able to use at least the last two (so right now Qt5.12/13) would be nice, but if there is a good reason to require Qt5.13 (is this is what 2.13 maps to) then I'm also ok with this, I just ask that it's documented in the commit WHY we made that choice.
Thanks